welcome_wizard/forms.py

```diff
@@ -6,7 +6,31 @@
 from welcome_wizard import models
 
 
-class ManufacturerImportFilterForm(NautobotFilterForm):  # pylint: disable=too-many-ancestors
+class ManufacturerImportForm(NautobotModelForm):  # pylint: disable=too-many-ancestors
+    """ManufacturerImport creation/edit form."""
+
+    class Meta:
+        """Meta attributes."""
+
+        model = models.ManufacturerImport
+        fields = "__all__"
+
+
+class ManufacturerImportBulkEditForm(TagsBulkEditFormMixin, NautobotBulkEditForm):  # pylint: disable=too-many-ancestors
+    """ManufacturerImport bulk edit form."""
+
+    pk = forms.ModelMultipleChoiceField(queryset=models.ManufacturerImport.objects.all(), widget=forms.MultipleHiddenInput)
+    description = forms.CharField(required=False)
+
+    class Meta:
+        """Meta attributes."""
+
+        nullable_fields = [
+            "description",
+        ]
+
+
+class ManufacturerImportFilterForm(NautobotFilterForm):
```

Comment:
Revert, this app doesn't have create/edit functionality for this model.
```suggestion
class ManufacturerImportFilterForm(NautobotFilterForm):  # pylint: disable=too-many-ancestors
```